client/src/components/Complete/index.tsx

```diff
@@ -26,11 +33,13 @@ const SignUpComplete = () => {
       </Fade>
       <Fade appear in timeout={1000}>
         <Box marginTop={6} display="flex" justifyContent="center">
-          <BWButton>View Upcoming Events</BWButton>
+          <BWButton component={Link} to="/events">
```

Comment:
We can use the `ROUTES` enum here. I just swapped the Header Menu links to use them 🚀

Reply:
Updated!